docs/DeveloperGuide.adoc

```diff
@@ -493,340 +503,226 @@ Here are the steps to create a new release.
-+
-****
-* Hint
-** Add the API method in link:{repoURL}/src/main/java/seedu/address/storage/AddressBookStorage.java[`AddressBookStorage`] interface.
-** Implement the logic in link:{repoURL}/src/main/java/seedu/address/storage/StorageManager.java[`StorageManager`] and link:{repoURL}/src/main/java/seedu/address/storage/XmlAddressBookStorage.java[`XmlAddressBookStorage`] class.
-* Solution
-** See this https://github.com/se-edu/addressbook-level4/pull/594[PR] for the full solution.
-****
+== Product Scope
 
-[[GetStartedProgramming-RemarkCommand]]
-=== Creating a new command: `remark`
+*Target user profile*:
+* has the need for flashcards
+* has a significant amount of material to memorise
+* have vast amount of different subjects to schedule
 
-By creating this command, you will get a chance to learn how to implement a feature end-to-end, touching all major components of the app.
+*Value proposition*: helps user memorise facts better.
 
-*Scenario:* You are a software maintainer for `addressbook`, as the former developer team has moved on to new projects. The current users of your application have a list of new feature requests that they hope the software will eventually have. The most popular request is to allow adding additional comments/notes about a particular contact, by providing a flexible `remark` field for each contact, rather than relying on tags alone. After designing the specification for the `remark` command, you are convinced that this feature is worth implementing. Your job is to implement the `remark` command.
+[appendix]
+== Product Scope
 
-==== Description
-Edits the remark for a person specified in the `INDEX`. +
-Format: `remark INDEX r/[REMARK]`
+*Target user profile*:
+* has the need for flashcards
+* has a significant amount of material to memorise
+* have vast amount of different subjects to schedule
 
-Examples:
+*Value proposition*: helps user memorise facts better.
 
-* `remark 1 r/Likes to drink coffee.` +
-Edits the remark for the first person to `Likes to drink coffee.`
-* `remark 1 r/` +
-Removes the remark for the first person.
+[appendix]
+== Feature Contribution
 
-==== Step-by-step Instructions
+=== Jethro Kuan
+Major enhancement: Implementing a graph structure to link Cards and Tags so that users are able to tag different cards
+in order to search them efficiently.
 
-===== [Step 1] Logic: Teach the app to accept 'remark' which does nothing
-Let's start by teaching the application how to parse a `remark` command. We will add the logic of `remark` later.
+Minor enhancement: Implementing a list card command so that users would be able to list the cards available
+for reference
 
-**Main:**
+=== Foo Yong Jie
+Major enhancement: Improving the UI / UX of the application, including implementing separate panels for tags, cards,
+and the display of cards.
+This allows users to use the application efficiently.
```

Comment:
This major enhancement is a bit vague. Maybe go into detail on what are the main changes you aim to make?